articles/integrations/aws-api-gateway/custom-authorizers/part-2.md

```diff
@@ -0,0 +1,262 @@
+    "Statement": [
+        {
+        "Effect": "Allow",
+        "Principal": {
+            "Service": [
+            "apigateway.amazonaws.com",
+            "lambda.amazonaws.com"
+            ]
+        },
+        "Action": "sts:AssumeRole"
+        }
+    ]
+    }
+    ```
+
+    ![](/media/articles/integrations/aws-api-gateway-2/part-2/pt2-9.png)
+
+    Click **Update Trust Policy**.
+
+9. You'll be redirected back to the **Summary** page. Copy down the **Role ARN** value for later use.
+
+![](/media/articles/integrations/aws-api-gateway-2/part-2/pt2-10.png)
+
+## Create the Lambda Function and Deploy the Custom Authorizer
+
+Now that you've configured your custom authorizer for your environment and tested it to see it works, you'll deploy it to AWS.
+
+1. First, you'll need to create a bundle that you can upload to AWS by running `npm run bundle`. This generates a `custom-authorizer.zip` bundle containing the source, configuration, and node modules required by AWS Lambda.
+
+2. Navigate to the [Lambda console](https://console.aws.amazon.com/lambda), and click **Create function**.
+
+![](/media/articles/integrations/aws-api-gateway-2/part-2/pt2-11.png)
+
+3. On the **Select blueprint** page, click **Author from scratch** to create a blank function.
+
+![](/media/articles/integrations/aws-api-gateway-2/part-2/pt2-12.png)
+
+4. On the **Configure triggers** page, click **Next** (you don't need to configure a trigger).
+
+![](/media/articles/integrations/aws-api-gateway-2/part-2/pt2-13.png)
+
+5. On the **Configure function** page, you'll provide all of the information needed for your new Lambda function. Under **Basic information**, provide values for the following parameters:
+
+| Parameter | Value |
+| - | - |
+| Name | A name for your Lambda function, such as `jwtRsaCustomAuthorizer` |
+| Description | A description for your Lambda function (optional) |
+| Runtime | Select `Node.js 4.3` |
+
+![](/media/articles/integrations/aws-api-gateway-2/part-2/pt2-14.png)
+
+a. Next, provide the function code. Under **Code entry type**, select **Upload a .ZIP file**. Click **Upload** and select the `custom-authorizer.zip` bundle you created earlier.
+
+b. Then, create the following three **Environment variables**. Note that this information is identical to that which is the `.env` file.
+
+| Parameter | Value |
+| - | - |
+| `TOKEN_ISSUER` | The issuer of the token. If Auth0 is the token issuer, use `https://${account.namespace}.auth0.com/` |
+| `JWKS_URI` | The URL of the JWKS endpoint. If Auth0 is the token issuer, use `https://${account.namespace}.auth0.com/.well-known/jwks.json` |
+| `AUDIENCE` | The token's `audience`. If Auth0 is the authorization server, the `audience` value is identical to the [API identifier](/apis#how-to-configure-an-api-in-auth0) |
+
+c. In the **Lambda function handler and role** section, set the following values:
+
+| Parameter | Value |
+| - | - |
+| Handler | `index.handler` |
+| Role | `Choose an existing role` |
```

Comment:
Is `Choose an existing role` an actual value or do you mean to say that the user should choose an existing role?